pkg/kubelet/eviction/eviction_manager_test.go

```diff
@@ -347,6 +377,109 @@ func TestMemoryPressure_VerifyPodStatus(t *testing.T) {
 	}
 }
 
+func TestPIDPressure_VerifyPodStatus(t *testing.T) {
+	testCases := map[string]struct {
+		wantPodStatus v1.PodStatus
+	}{
+		"eviction due to pid pressure": {
+			wantPodStatus: v1.PodStatus{
+				Phase:   v1.PodFailed,
+				Reason:  "Evicted",
+				Message: "The node was low on resource: pids. Threshold quantity: 1200, available: 500. ",
+			},
+		},
+	}
+	for name, tc := range testCases {
+		for _, enablePodDisruptionConditions := range []bool{true, false} {
+			t.Run(fmt.Sprintf("%s;PodDisruptionConditions=%v", name, enablePodDisruptionConditions), func(t *testing.T) {
+				defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.PodDisruptionConditions, enablePodDisruptionConditions)()
+
+				podMaker := makePodWithPIDStats
+				summaryStatsMaker := makePIDStats
+				podsToMake := []podToMake{
```

Comment:
I don't understand why this is failing..

So in this code, what are podToMake.name and podToMakePriority? You created empty arrays so are both just the nil values?

Reply:
Sorry, where do you see it failing? In the checks below I see all jobs succeeded except the pending `tide` one.

I ran the test only locally via ` go test -v ./pkg/kubelet/eviction` and those tests all passed for me.

Reply:
I meant I am surprised that these tests are not failing.

Reply:
It would be good to intialize this values with something other than nil and 0 for name/priority.